### PR TITLE
Typo in FAQ

### PR DESCRIPTION
**Pull request by ThatJSGuy**
"and become productive in for React developers"

changed the sentence to correct the typo and updated grammar for clarity.

# Overview

# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [x] Docs / tests / types / typos

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

# Use cases and why

- 1. One use case
- 2. Another use case

# Checklist:

- [x] My code follows the [developer guidelines of this project](https://github.com/BuilderIO/qwik/blob/main/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

File: packages/docs/src/routes/docs/(qwik)/faq/index.mdx [hunk 1]
```diff
@@ -55,7 +55,7 @@ The short answer is that Qwik solves a problem that other frameworks can't solve
 
 ## Is Qwik hard to learn?
 
-We designed Qwik to be [extremely easy to learn](/docs/(qwikcity)/guides/react-cheat-sheet/index.mdx) and become productive in for React developers. Developing components is pretty much the same as React, and routing is inspired by Nextjs and others.
+Qwik is designed with React developers in mind, ensuring it's [effortless to learn] (/docs/(qwikcity)/guides/react-cheat-sheet/index.mdx) and promotes rapid productivity. Developing components is pretty much the same as React, and routing is inspired by Nextjs and others.
```
Comment (shairez) [inline packages/docs/src/routes/docs/(qwik)/faq/index.mdx:58]: I would make it even more generic saying something like: 
"Qwik is designed with React (and other JSX based frameworks) in mind, ensuring..."

**Comment by shairez** [comment]
Thanks @ThatJSGuy !
added a small comment

File: packages/docs/src/routes/docs/(qwik)/faq/index.mdx (hunk 1 above)
Comment (ThatJSGuy) [reply packages/docs/src/routes/docs/(qwik)/faq/index.mdx]: I went ahead and changed this to your reworded version :)

**Comment by shairez** [comment]
thanks @ThatJSGuy !

**Pull request merged**